January_25/MainWindow.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<Window x:Class="January_25.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:local="clr-namespace:January_25"
+        xmlns:viewModel="clr-namespace:January_25.ViewModels"
+        xmlns:view="clr-namespace:January_25.Views"
+        mc:Ignorable="d"
+        Title="MainWindow" Height="300" Width="600">
+    <Window.DataContext>
+        <viewModel:MainWindowViewModel/>
+    </Window.DataContext>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition/>
+            <ColumnDefinition/>
+        </Grid.ColumnDefinitions>
+
+        <ListView x:Name="mainListView" Margin="10" Grid.Column="0" HorizontalContentAlignment="Stretch"
+                  ItemsSource="{Binding Path=Customers}" 
+                  SelectedItem="{Binding Path=SelectedCustomer}"
+                  />
```

Comment:
Please put this on the previous line.

Reply:
sorry didn't catch - this /> to put?

Reply:
Closed part of the tag should be placed on the same line where the last property is declared.